### PR TITLE
[WIP] fixing up the nav bar, logistics page, reflecting changes for virtual event

### PR DESCRIPTION
**Pull request by llevitis**
This PR ended up being less atomic than planned. It's currently 
- [x] updating the height of the nav bar and making it be white before scrolling as well
- [x] updated about us for team members
- [x] logistics page is in progress
- [x] update role description
- [ ] add mattermost and crowdcast images

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -217,7 +217,9 @@ showSessions: true
...
+aboutUs: "The OHBM Open Science SIG mission is to advance neuroimaging research by fostering the open sharing of ideas, data, and tools between members of the OHBM community. This will be accomplished by organizing educational events, supporting collaborative initiatives, and providing mentorship to junior researchers.
+As with previous years, our three main contributions to the OHBM annual meetin are: 1) organize a Hackathon to precede the annual meeting, 2) introduce the tools of open science via interactive tutorials as part of the TrainTrack, and 3) facilitate collaborative work on a wide range of open science projects.
```
Comment (Remi-Gau) [inline _config.yml:221]: if we find a find a way to render those as a numbered list it would be nice.

But really low priority ATM

**Comment by Remi-Gau** [comment]
I get some errors that some images are missing but I think that either this should be fixed when we merge into master.

I guess you could also rebase or merge master into this branch to fix this.

```
/hackathon2020/img/about-section/undraw/undraw_share_link_qtxe.png' not found.
```

File: _data/logistics.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+-
```
Comment (Remi-Gau) [inline _data/logistics.yml:1]: We don't need the `_data/logistics.yml` file anymore, right?

File: logistics.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+* Americas Hub - Hosts: Liza Levitis 
+* Europe, Middle East, and Africa Hub - Hosts: Remi Gau
+* Asia and Pacific Oceania Hub - Hosts: Angela Tam and Chao-Gan Yan
+{% endcollapse %}
+
+{% collapse Hub structure and general organization %} 
+In order for the event to run as smoothly as possible across all hubs, each hub will have a list of roles filled by volunteers. 
+Some of the roles are selected by the OS-SIG while others are open to volunteers.
+
+##### Selected by the OS-SIG
```
Comment (Remi-Gau) [inline logistics.md:24]: Do we want to mention which ones are selected by us or do we only want to have those we people can "volunteer" for?

File: logistics.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+{% collapse Hackathon hub hosts %}
+Each hub has two hackathon hosts who welcome the participants during the kickoff and project pitches, moderate the unconferences, and liase with the other volunteers.
```
Comment (Remi-Gau) [inline logistics.md:26]: As Camille suggested I think we need to rework those role description but I think we need to merge this PR first given how many things it touches (and then we will rework the wording).

File: logistics.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+
+#### HackTrack
+{% accordion HackTrack %} 
+As with previous OHBM hackathons, we've developed a project template that we would like project leads to use for submitting 
+their project proposals.  
+{% collapse Project submission %}
+Each project is allocated to a project helper who will follow the project from submission until the end of the hackathon.
+To make sure that project helpers have enough time to review project submissions, we're setting two deadlines: 
+1. 2 weeks before the hackathon: submit a draft for project helpers to give feedback on
```
Comment (Remi-Gau) [inline logistics.md:87]: We might want to change this to an unordered list: rendering looks weird, no?

1. 2 weeks... 
2. 1 week...

File: logistics.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+2. 1 week before hackathon: hard deadline
+
+[**Project checklist**](https://docs.google.com/spreadsheets/d/1SY-Hnhc-bPX0RA47icVukUVVJ-BrX9ykPFQUBY-U1go/edit#gid=0): this is a spreadsheet to help the project helpers making sure that each project is on track and submitted everything.
```
Comment (Remi-Gau) [inline logistics.md:90]: Should we change this to a "view only" link? Not that I think that a random person is going to want to come and change our spreadsheet but...

File: logistics.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+#### Platforms
+
+{% accordion software %}
+{% collapse Crowdcast %}
+Each hub will be using Crowdcast for the main sessions, and this content will be streamed on Youtube simultaneously such that those who are not formally registered can follow along as well.
+{% endcollapse %} 
+
+{% collapse Mattermost %}
+* Generic channels:
```
Comment (Remi-Gau) [inline logistics.md:66]: To do later if we have time: add images of the mattermost and crowdcast logo.

File: logistics.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+{% accordion TrainTrack %}
+{% collapse General computing requirements %}
+In addition to some software that we will list in the subsequent section, these are the general computing requirements: 
+1. Administrative access to your computer such that you can install software on your own. 
+2. If you're using Windows, the required version is Windows 10. 
+{% endcollapse %}
+
+{% collapse Required software %}
+Along with the platforms that we will be using for virtual TrainTrack presentations and communication, we ask that you
```
Comment (Remi-Gau) [inline logistics.md:115]: Maybe  we will need to mention that this is i they want to attend the whole train track and that the requirement may vary.

Can be updated later.

Comment (llevitis) [reply logistics.md:115]: That's true - we could modify the installation verification script as well based on a session(s), but I do think for now we could leave it as is for simplicity

File: _data/logistics.yml (hunk 2 above)
Comment (llevitis) [reply _data/logistics.yml:1]: correct - will remove it

File: logistics.md (hunk 4 above)
Comment (llevitis) [reply logistics.md:26]: agreed (my bad again for this monstrosity of a PR lol)

File: logistics.md (hunk 5 above)
Comment (llevitis) [reply logistics.md:87]: oui, good point!

File: logistics.md (hunk 6 above)
Comment (llevitis) [reply logistics.md:90]: yes, definitely!

Comment (llevitis) [reply logistics.md:90]: hmm seems like changing it to a view only link will modify permissions for everyone, so we may have to duplicate the link

**Comment by llevitis** [comment]
> I have several comments and I think we could try to tweak the CSS to make it prettier and so on but we can worry about all that later.

@Remi-Gau what specifically would you like changed aesthetically? if it's simple, i could try doing it today or tomorrow so that we can merge this asap

File: logistics.md (hunk 4 above)
Comment (Remi-Gau) [reply logistics.md:26]: you got plenty of help from me. :-)

**Comment by Remi-Gau** [comment]
> > I have several comments and I think we could try to tweak the CSS to make it prettier and so on but we can worry about all that later.
> 
> @Remi-Gau what specifically would you like changed aesthetically? if it's simple, i could try doing it today or tomorrow so that we can merge this asap

I was thinking of adding a bit of padding above each `section` (hubs, platform, hacktrack...) to make it easier to visualize them but as this is markdown, I am not sure how to do this now.

**Comment by Remi-Gau** [comment]
oh and merge conflicts...

**Comment by Remi-Gau** [comment]
> > > I have several comments and I think we could try to tweak the CSS to make it prettier and so on but we can worry about all that later.
> > 
> > 
> > @Remi-Gau what specifically would you like changed aesthetically? if it's simple, i could try doing it today or tomorrow so that we can merge this asap
> 
> I was thinking of adding a bit of padding above each `section` (hubs, platform, hacktrack...) to make it easier to visualize them but as this is markdown, I am not sure how to do this now.

Actually I tried putting the whole page into a div element and it works except that now you have to reformat the markdown headings into HTML headings. It could help with formatting.

Let me know if you want me to push that.

```
---
layout: post
title: Get ready for the first virtual OHBM Hackathon!
isStaticPost: true
permalink: /logistics/
---
<div class="col-md-8">

...

{% endaccordion%}

</div>
````

**Comment by Remi-Gau** [comment]
@llevitis made a couple of changes.
The css needs would need to be refactored but I have no idea how to do this on a .md file...

**Comment by llevitis** [comment]
> @llevitis made a couple of changes.
> The css needs would need to be refactored but I have no idea how to do this on a .md file...

shall checkout your changes now! @Remi-Gau

**Comment by llevitis** [comment]
Merging this everything but the kitchen sink PR with Remi's approval!

**Pull request merged**